SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -149,21 +157,72 @@ public StatusResponse GetStatus()
         }
 
         /// <summary>
-        /// Mutes Spotify (Requires nircmd.dll)
+        /// Mutes Spotify (Requires Windows 7 or newer)
         /// </summary>
         public void Mute()
         {
-            if (File.Exists("nircmd.dll"))
-                DoNirCmd("muteappvolume spotify.exe 1");
+            //Windows < Windows Vista Check
+            Contract.Requires(Environment.OSVersion.Version.Major >= 6);
```

Comment:
So, after reading a bit about Contracts, I'm unsure if it's worth using here while we still have some normal "if-throw-else"-statements. So why did you chose this over "if-throw-else"-statements? Any particular reason?